src/utils/jellyfin-apiclient/ConnectionState.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Server state values for a connected server used by jellyfin-apiclient.
+ */
+export enum ConnectionState {
```

Comment:
I'm not sure that `utils` is the best place for this, but we really don't have a dedicated place for constants of this type. :man_shrugging: